specs/savings_account_spec.rb

```diff
@@ -20,39 +12,56 @@
     end
 
     it "Requires an initial balance of at least $10" do
-      # TODO: Your test code here!
+      proc{ account = Bank::SavingsAccount.new(1212, 9)
+      }.must_raise(ArgumentError)
     end
   end
-
+  
   describe "#withdraw" do
     it "Applies a $2 fee each time" do
-      # TODO: Your test code here!
+      savings_account = Bank::SavingsAccount.new(11, 200)
+      new_balance = savings_account.withdraw(10)
+      new_balance.must_equal 188
```

Comment:
If you run the test

`savings_account.balance.must_equal 188`

here, it will fail.  It's because the balance isn't getting updated.  when you call super in the savings account, you should be passing it the `amount  + 2` to ensure it adds in the fee.  You're only getting the return value from the withdraw method not the balance attribute.